examples/thrift/build.gradle

```diff
@@ -0,0 +1,56 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
```

Comment:
This comment isn't quite right because this is isn't a top-level build file :). I'm okay if it is just deleted.